SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.iOS/CrashesExtension.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Azure.Mobile.Crashes.iOS.Bindings;
+
+namespace Microsoft.Azure.Mobile.Crashes
+{   
+    public static partial class Crashes
```

Comment:
You don't need that class and reflection. Just move this call in static constructor of PlatformCrashes.